cmd/neofs-node/config/contracts/config.go

```diff
@@ -0,0 +1,67 @@
+package contractsconfig
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+)
+
+const (
+	subsection = "contracts"
+)
+
+// Netmap returns value of "netmap" config parameter
```

Comment:
doesnt it needed to say that it tries to convert it to custom type but not just return value from config?

Reply:
Function return type says for itself. Also it will be mentioned, that panic is thrown if it is not 20-byte LE hex-encoded string.